features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,30 @@
+Given("I am on the landing page") do
+  visit '/'
+  end
+  
+  When("i click on the link {string}") do |login|
+    click_on(login)
+  end
+  
+  When("I fill in {string} with {string}") do |a, b|
```

Comment:
Please do not use single letter variable names. IMO this should be `field, content`